Utils/Dataflow/019_esFormat/esFormat.php

```diff
@@ -41,18 +93,53 @@ function convertIndexToLowerCase(&$a) {
 }
 
 function getAction($row) {
-  global $DEFAULT_ACTION;
+  /* Determine action for the document.
+
+  Action is 'update' if {'_update': true} key-value pair is 
+  present in the document. Otherwise, default value is returned.
```

Comment:
I'm afraid that after 67a80d8 (#320) this description is incomplete.
I can merge this PR to immediately fix it in the next one, or you can merge `master` here and update the description accordingly.

Reply:
Done, 3a078d4.